invenio_rdm_records/oai.py

```diff
@@ -27,6 +30,46 @@ def dublincore_etree(pid, record):
     return DublinCoreXMLSerializer().serialize_object_xml(record["_source"])
 
 
+def datacite_etree(pid, record):
```

Comment:
I think it's time we also add some test coverage for this file in general... I didn't find any tests for this part, so if that's really the case, you can kick-off a `tests/tests_oai.py` :)

Reply:
done